e2e/cloudfront/cloudfront_test.go

```diff
@@ -98,6 +121,52 @@ var _ = Describe("CloudFront", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("updating s3 bucket policy should succeed", func() {
+			out, err := cli.EnvShow(&client.EnvShowRequest{
+				AppName: appName,
+				EnvName: "test",
+			})
+			Expect(err).NotTo(HaveOccurred())
+			var accountID string
+			var cfDistID string
+			for _, resource := range out.Resources {
+				switch resource["logicalID"] {
+				case "PublicLoadBalancer":
+					parsed, err := arn.Parse(resource["physicalID"])
+					Expect(err).NotTo(HaveOccurred())
+					accountID = parsed.AccountID
+				case "CloudFrontDistribution":
+					cfDistID = resource["physicalID"]
+				}
+			}
+			Expect(accountID).ToNot(BeEmpty())
+			Expect(cfDistID).ToNot(BeEmpty())
+			_, err = s3Client.PutBucketPolicy(&s3.PutBucketPolicyInput{
```

Comment:
I see, I assume we will add this piece of info into our documentation in the env manifest right?

Reply:
yeah it's in my doc pr.